doc/source/drivers/raster/mem.rst

```diff
@@ -64,6 +64,15 @@ or
 -  GEOTRANSFORM: Set the affine transformation coefficients. 6 real
    numbers with '/' as separator (optional)
 
+Coordinate reference system
```

Comment:
I would not create a new paragraph for that, but included it in the above list. The mention of the version can be done with something like ``SPATIALREFERENCE: (GDAL >= 3.7) Set the projection...``

Reply:
excellent, will do - there are also undocumented elements I'll explore and test them also